src/Data/ProgrammesDb/EntityRepository/CoreEntityRepository.php

```diff
@@ -203,6 +203,47 @@ public function findByIds(array $ids): array
             ->getQuery()->getResult(Query::HYDRATE_ARRAY);
     }
 
+    /**
+     * @param int[]    $ancestryDbIds
+     * @param string   $entityType
+     * @param int|null $limit
+     * @param int      $offset
+     * @return array
+     */
+    public function findDescendantsByType(array $ancestryDbIds, string $entityType, ?int $limit, int $offset) : array
```

Comment:
Should this have a space before the colon? E.G. `): array` instead of `) : array`. If so, should that be added to our codesniffer?